frontend/projects/valtimo/layout/src/lib/components/widget-field/widget-field.component.scss

```
@@ -23,7 +23,15 @@
     justify-content: space-between;
   }
 
-  &,
+  & {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    width: 100%;
+    min-height: var(--widget-available-height, auto);
+    box-sizing: border-box;
+  }
```

Stylelint (17.4.0)

[error] Unexpected redundant nested style rule (block-no-redundant-nested-style-rules)

(block-no-redundant-nested-style-rules)